scenedetect/scene_manager.py

```diff
@@ -136,6 +140,96 @@ def write_scene_list(output_csv_file, scene_list, cut_list=None):
             '%d' % duration.get_frames(), duration.get_timecode(), '%.3f' % duration.get_seconds()])
 
 
+def write_scene_list_html(output_html_filename, scene_list, cut_list=None, css=None, css_class='mytable',
```

Comment:
That makes sense. I put it there because this is also where the `write_scene_list` function is that writes the csv file. I basically tried to make these two functions the same. One minor thing is that the `write_scene_list` function takes a file handle, but the html version just takes a filepath. This is because the `simpletable` code does the opening of the file and I thought it easier to have this discrepancy than rewriting the `simpletable` code at the time. If these two functions get consolidated, it might make sense to standardize the file handling as well.

Reply:
Good points @wjs018, I'll make a note to see if I can modify write_scene_list to also just take the output file path as a string, instead of a file handle, to make it consistent.  Also good point in that write_scene_list and write_scene_list_html belong together - perhaps I will move them into a new sub-module (e.g. `export.py` or `scene_export.py` - I'm not sure which is better, or if there's a better name - feel free to choose or suggest one!).